### PR TITLE
Header dropdown

### PR DESCRIPTION
**Pull request by josephj**
* Refactored the prev/next arrows which uses a lot of `position:absolute`
* Will update URL and state for James to have further implementation

### CODE REVIEW

**Comment by josephj** [comment]
[(1) Multiple Service - Header dropdown menu for switching different services [0]](https://trello.com/c/ihyMjEEZ/98-1-multiple-service-header-dropdown-menu-for-switching-different-services-0)

File: client/src/components/NavBar.js [hunk 1]
```diff
@@ -32,30 +42,18 @@ const Wrapper = styled.div`
...
+  vertical-align: middle;
   width: 32px;
 `;
-const Title = styled.a.attrs({ href: '/' })`
+const Title = styled.span`
   display: inline-block;
   font-size: 18px;
   font-weight: bold;
+  position: relative;
+  z-index: 3;
```
Comment (james1239090) [inline client/src/components/NavBar.js]: why here need to change z-index ?

Comment (josephj) [reply client/src/components/NavBar.js]: Thanks, I will remove this.

File: client/src/components/DateBar.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      </Arrow>
+      {startMonth} - {endMonth} {year}
+      <Arrow onClick={onNextClick}>
+        <img src={rightArrowIcon} role="presentation" />
+      </Arrow>
+    </Wrapper>
+  );
+};
+
+const Wrapper = styled.h1`
```
[the remark this answers is not in the document]
Comment (josephj) [reply client/src/components/DateBar.js]: Thanks, I will change to `<div/>`

**Pull request merged**